packages/app/components/Settings/SettingsPage.tsx

```diff
@@ -82,24 +88,46 @@ export default function SettingsPage({
               />
             )}
             <Upload
-              style={{ marginBottom: "60px" }}
               action={"/api/v1/profile/upload_picture"}
               beforeUpload={beforeUpload}
               showUploadList={false}
               onChange={(info) => {
-                info.file.status === "uploading"
-                  ? setUploading(true)
-                  : setUploading(false);
+                setUploading(info.file.status === "uploading");
                 mutate();
               }}
             >
               <Button
                 icon={<UploadOutlined />}
-                style={{ marginBottom: "60px" }}
+                style={{
```

Comment:
buttons were of different widths based on text size -- this changed that

Reply:
pls make these into styled components if they have the same width